ui/app/pages/confirm-add-suggested-token/confirm-add-suggested-token.component.js

```diff
@@ -45,6 +48,16 @@ export default class ConfirmAddSuggestedToken extends Component {
           <div className="page-container__subtitle">
             { this.context.t('likeToAddTokens') }
           </div>
+          { hasTokenDuplicates ?
+            (<div className="warning">
```

Comment:
The linter doesn't like these lines either: https://app.circleci.com/jobs/github/MetaMask/metamask-extension/137109

The closing paren needs to match the indentation of the opening paren, and the `div` component should be on a new line.

Reply:
Made a change, hoping this works. JSX linting is kinda new to me.